components/typography/typography.html

```diff
@@ -0,0 +1,162 @@
+                <h1 class="text-24 mt-1">Font Sizes</h1>
+                <div class="flex flex-col align-items-baseline width-max m-1">
+
+                    <div class="font-sizes-container">
+                        <p class="text-12">Hi</p>
+                        <p class="text-14">Hi</p>
+                        <p class="text-16">Hi</p>
+                        <p class="text-18">Hi</p>
+                        <p class="text-24">Hi</p>
+                        <p class="text-36">Hi</p>
+                        <p class="text-48">Hi</p>
+                        <p class="text-64">Hi</p>
+                        <p class="text-72">Hi</p>
+                        <p class="text-96">Hi</p>
+                        <p class="text-112">Hi</p>
+                        <p class="text-144">Hi</p>
+                    </div>
+
+                </div>
+
+                <div class="code-snippet-container">
+                        <script type="text/plain" class="line-numbers language-markup">
+                            <p class="text-12">Hi</p>
+                            <p class="text-14">Hi</p>
+                            <p class="text-16">Hi</p>
+                            <p class="text-18">Hi</p>
+                            <p class="text-24">Hi</p>
+                            <p class="text-36">Hi</p>
+                            <p class="text-48">Hi</p>
+                            <p class="text-64">Hi</p>
+                            <p class="text-72">Hi</p>
+                            <p class="text-96">Hi</p>
+                            <p class="text-112">Hi</p>
+                            <p class="text-144">Hi</p>
+                        </script>
+                             
+
+                </div>
+
+            </div>
+
+
+
+
+            <div class="docs-content-sub-container">
+                <h1 class="text-24 mt-1">Text Alignment</h1>
+                <div class="flex flex-col align-items-baseline width-max m-1">
+                    <div class="text-alingment-container">
+                        <p class="text-left">Left Text</p>
+                        <p class="text-center">Center Text</p>
+                        <p class="text-right">Right Text</p>
+                    </div>
+                </div>
+
+                <div class="code-snippet-container">
+                    <script type="text/plain" class="line-numbers language-markup">
+                        <p class="text-left">Left Text</p>
+                        <p class="text-center">Center Text</p>
+                        <p class="text-right">Right Text</p>
+                    </script>
+                </div>
+            </div>
+
+
+            <div class="docs-content-sub-container">
+                <h1 class="text-24 mt-1">Text Decoration</h1>
+                <div class="flex flex-col align-items-baseline width-max m-1">
+                    <div class="flex text-16 text-decoration-container">
```

Comment:
suggestion: short classnames
for future

Reply:
Okay Anjali